build.sbt

```diff
@@ -4,22 +4,32 @@ name := "safe-config"
 organization := "com.kinja"
 version := "1.1.2-SNAPSHOT"
 
-scalaVersion := "2.12.8"
-crossScalaVersions := Seq("2.12.8", "2.11.12")
+scalaVersion := "2.13.0"
+crossScalaVersions := Seq("2.13.0", "2.12.8")
 
 scalacOptions ++= Seq(
   "-deprecation",          // Show details of deprecation warnings.
   "-encoding", "UTF-8",    // Set correct encoding for Scaladoc.
   "-feature",              // Show details of feature warnings.
   "-unchecked",            // Show details of unchecked warnings.
-  "-Xfatal-warnings",      // All warnings should result in a compiliation failure.
-  "-Xfuture",              // Disables view bounds, adapted args, and unsound pattern matching in 2.11.
   "-Xlint",                // Ensure best practices are being followed.
-  "-Yno-adapted-args",     // Prevent implicit tupling of arguments.
   "-Ywarn-dead-code",      // Fail when dead code is present. Prevents accidentally unreachable code.
   "-Ywarn-value-discard"   // Prevent accidental discarding of results in unit functions.
 )
 
+scalacOptions ++= (CrossVersion.partialVersion(scalaVersion.value) match {
+  case Some((2, scalaMajor)) if scalaMajor >= 13 =>
+    Seq(
+      "-Ymacro-annotations"
+    )
+  case _ =>
+    Seq(
+      "-Xfatal-warnings",  // All warnings should result in a compiliation failure.
```

Comment:
`-Xfatal-warnings` can't be enabled yet because the old Java converters API is deprecated but the new one is not present in 2.12 so inevitably we'll have deprecation warnings.

Reply:
There are a lot more flags in some other projects: https://github.com/gawkermedia/kinja-core/blob/master/build.sbt#L70-L110 — would it make sense to bring some of them here?